src/ui/BookDetail/BookCommentList.tsx

```diff
@@ -58,42 +61,46 @@ const BookCommentList = ({ bookId }: Props) => {
       );
   }, [bookCommentsQueryInfo, bookId]);
 
+  const handleCommentError = (error: unknown) => {
+    if (!isAxiosErrorWithCustomCode(error)) {
+      return;
```

Comment:
👍 깔끔하네요! 유틸 함수로 분리해서 범용적으로 사용해도 좋을 것 같아요
 '요청을 처리하지 못했어요. 잠시 후 다시 시도해주세요.' 같이 공통으로 사용할 수 있는 메시지를 하나 만들고, 만약 커스텀 코드가 없는 경우에 노출시켜줘도 될 것 같아요! 

```suggestion
      showToast({ defaultMessage, duration: 3000 });
      return;
```